src/cmds/core/other.py

```diff
@@ -25,7 +25,14 @@ async def no_hints(
             "Once the event is over you are more then welcome to share solutions/write-ups/etc and try them in the "
             "After Party event."
         )
-
+    @slash_command(guild_ids=settings.guild_ids, description="A simple reply proving a link to the support desk article on how to get support")
+    async def support(
+        self, ctx: ApplicationContext
+    ) -> Message:
+        """A simple reply proving a link to the support desk article on how to get support"""
+        return await ctx.channel.send(
+            "https://help.hackthebox.com/en/articles/5986762-contacting-htb-support"
+        )
```

Comment:
I would add a cooldown to avoid getting spammed.
```suggestion
    @slash_command(guild_ids=settings.guild_ids, description="A simple reply proving a link to the support desk article on how to get support")
    @cooldown(1, 60, commands.BucketType.user)
    async def support(
        self, ctx: ApplicationContext
    ) -> Message:
        """A simple reply proving a link to the support desk article on how to get support"""
        return await ctx.channel.send(
            "https://help.hackthebox.com/en/articles/5986762-contacting-htb-support"
        )
```